bubblewrap.c

```diff
@@ -482,9 +482,16 @@ acquire_privs (void)
         die_with_error ("Unable to set fsuid");
 
       /* setfsuid can't properly report errors, check that it worked (as per manpage) */
-      new_fsuid = setfsuid (-1);
-      if (new_fsuid != real_uid)
-        die ("Unable to set fsuid (was %d)", (int)new_fsuid);
+      new_fsid = setfsuid (-1);
+      if (new_fsid != real_uid)
+        die ("Unable to set fsuid (was %d)", (int)new_fsid);
+
+      if (setfsgid (real_gid) < 0)
+        die_with_error ("Unable to set fsgid");
```

Comment:
I don't understand this. The binary is not setgid, only setuid, so it should have the right gid always. What exactly happens that make this a problem?